Added Gyre-Carlin

Noikar opened the pull request:
<!-- place 'x' mark between square [] brackets to affirm: -->
**_I affirm:_**
- [X] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [X] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Noikar committed:
Added Gyre-Carlin

cocosolos commented on scripts/zones/PsoXja/mobs/Diremite.lua:
```diff
@@ -0,0 +1,12 @@
+end;
+
+function onMobDespawn(mob)
+    tpz.mob.phOnDespawn(mob, ID.mob.GYRE_CARLIN_PH, 5, math.random(3600*4, 3600*5)) -- 4-5 hours.
```
Where did you get 4-5 hours? I'm only seeing as low as 15-30 mins on wikia, and again I don't think lottery mobs should have an upper bound on their spawn time.

Noikar replied:
Oh sorry, I thought I pasted the clean version not the Dawnbreak version.

cocosolos commented on scripts/zones/PsoXja/IDs.lua:
```diff
@@ -31,6 +31,16 @@ zones[tpz.zone.PSOXJA] =
     {
+        GYRE_CARLIN_PH =
+        {
+            [16814317] = 16814320, -- -66.201 -0.352 189.181
+            [16814318] = 16814320, -- -56.059 -0.352 186.055
+            [16814324] = 16814331, -- -13.760 -0.352 224.323
+            [16814325] = 16814331, --  -6.011 -0.352 205.851
+            [16814328] = 16814344, --   9.065 -0.352 168.035
+            [16814329] = 16814344, --  34.530 -0.352 165.232
+            [16814330] = 16814344, --  34.057 -0.352 165.306
```
It seems that none of these NM IDs exist. Also the only placeholder seems to be 16814330 and is supposed to spawn in a few different locations (there's only 1 room with 3 Diremites, the 3rd one in that room is 16814330 and is supposed to spawn in a few different places.)

cocosolos replied:
Sorry I might actually be wrong about this, they're there they just don't have any spawn points so they're not being loaded at all. Same with the other PHs, looks like there are multiples (assuming only 1 is supposed to be up at a time) but none of the other ones have spawn points set.

cocosolos replied:
PHs should be:
```lua
[16814319] = 16814320,
[16814330] = 16814331,
[16814343] = 16814344,
```

Noikar committed:
Update Diremite.lua

The 15-30 minutes was flagged as "verification needed" on the wiki so we added our own timer. I've adjusted it to 30 minutes for you.

Noikar committed:
Update Diremite.lua

Noikar committed:
Update IDs.lua

I'll take your word for it.

Noikar committed:
Updated NM spawn point

Noikar committed:
Update IDs.lua

Noikar committed:
Update Diremite.lua

Noikar committed:
Update Diremite.lua

Removed semicolons
